scripts/netContain/Dockerfile

```diff
@@ -17,30 +17,18 @@
 
 FROM ubuntu:16.04
```

Comment:
This is something we can discuss later. I don't plan to convert this Dockerfile to something else in this PR.